lms/djangoapps/courseware/access.py

```diff
@@ -325,10 +351,10 @@ def can_see_in_catalog():
         In this case we use the catalog_visibility property on the course descriptor
         but also allow course staff to see this.
         """
-        return (
+        return ACCESS_GRANTED if (
             course.catalog_visibility == CATALOG_VISIBILITY_CATALOG_AND_ABOUT or
```

Comment:
And then you can update the code here and below so it's not wrapped with the `if` statements.